src/components/RichMarker.tsx

```diff
@@ -166,3 +168,283 @@ export function MarkerIcon({
     </>
   );
 }
+
+export function ObjectMarker({
```

Comment:
super že je to v súbore `RichMarker` ale myslel som zaintegrovať priamo do `RichMarker`/`MarkerIcon` komponentu.

Reply:
spravil som to v https://github.com/FreemapSlovakia/freemap-v3-react/commit/7fe519486ab681878da30f61c94e42c77264f688